### PR TITLE
Option to add and annotate namespaces for openshift-service-catalog

### PR DESCRIPTION
**Pull request by apoorva11029**
Adding tasks in roles/aci to create and annotate the following namespaces:
openshift-console
openshift-monitoring
openshift-web-console
kube-service-catalog

### CODE REVIEW

File: roles/aci/tasks/main.yml [hunk 1]
```diff
@@ -9,3 +9,20 @@
...
+  include_role:
+    name: lib_openshift
+- name: Create namespaces
+  oc_project:
+    state: present
+    name: "{{ item }}"
+  with_items: "{{ annotate_namespace_list }}"
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+- name: Get aci policy tenant name
+  shell:  "grep -m1 'aci-policy-tenant' {{ aci_deployment_yaml_file }} | awk -F '\"' '{print $4}'"
```
Comment (patrickdillon) [inline roles/aci/tasks/main.yml]: For parsing yaml files use `yedit`.  Example [provided by Vadim](https://github.com/openshift/openshift-ansible/pull/11439/files#r271328607)
```
- name: Read master storage backend setting
    yedit:
      state: list
      src: /etc/origin/master/master-config.yaml
      key: kubernetesMasterConfig.apiServerArguments.storage-backend
    register: _storage_backend
```

Comment (apoorva11029) [reply roles/aci/tasks/main.yml]: Seems like there's an issue using yedit in this context, source: https://github.com/openshift/openshift-ansible/pull/11439/files#r271760006

Comment (patrickdillon) [reply roles/aci/tasks/main.yml]: Thanks for looking into that.

**Comment by apoorva11029** [comment]
@patrickdillon done, squashed the commits.

**Pull request merged**